src/components/Brand.tsx

```diff
@@ -0,0 +1,25 @@
+import Logo from "@/components/Logo";
+
+interface BrandProps {
+  className?: string;
+  variation?: "desktop" | "mobile";
+}
+
+export default function Brand({
+  className = "",
+  variation = "desktop",
+}: BrandProps) {
+  const isMobile = variation === "mobile";
+
+  return (
+    <div className={`flex items-center gap-0 ${className}`}>
```

Comment:
```suggestion
    <div className={twMerge("flex items-center gap-0", className)}>
```
the "mb-10" from the div that I suggested removing in roomId page.tsx will simply be appended here saving us one level of unnecessary div I think